src/cli/tc.rs

```diff
@@ -28,7 +28,14 @@ pub async fn run() -> Result<()> {
       start_server(config).await?;
       Ok(())
     }
-    Command::Check { file_path, n_plus_one_queries, schema } => {
+    Command::Check { file_path, n_plus_one_queries, schema, operation } => {
+      let operation = operation
+        .iter()
+        .flat_map(|operation| operation.split(',').map(|operation| operation.to_string()))
+        .collect::<Vec<String>>();
+
+      println!("Operation: {:?}", operation);
```

Comment:
@tusharmath, how can I use the queries now?

also, can you provide some queries that I can run against jsonplaceholder? Thanks!

Reply:
- You will need to parse the graphQL file to identify the queries.
- Sure: Something like should be a valid graphQL query `query {posts {user {name}}}`